edgelet/contrib/debian8/postinst

```diff
@@ -10,9 +10,13 @@ case "$1" in
             chown ${USER}:${GROUP} /var/lib/iotedge
             chown ${USER}:${GROUP} /var/log/iotedge
             chown ${USER}:${GROUP} /etc/iotedge/config.yaml
```

Comment:
Your `debian/postinst` change replaces this line with a `chown` statement for `config.yaml.template`. Why did you not do the same thing here (for `debian8`)?

Reply:
Good catch. That was an error. I missed that test case for Deb8.